examples/kv-server/pom.xml

```diff
@@ -48,24 +48,20 @@
                     <webAppXml>${project.basedir}/src/main/webapp/WEB-INF/jetty-web.xml</webAppXml>
                     <webApp>
                         <contextPath>/api</contextPath>
+                        <extraClasspath>../../core/target/classes;../../containers/jersey2-routing/target/classes</extraClasspath>
```

Comment:
The config specify the extra class to load from instead of the dependency jar file.
and if there is any changes of the class, it will be automatically reloaded by jetty.